haproxy.tmpl

```diff
@@ -3,12 +3,14 @@ global
     tune.ssl.default-dh-param 2048{{.ExtraGlobal}}
 
 defaults
-    mode    http
+    mode    {{.ProxyMode}}
```

Comment:
Let's keep this as http, since the reqMode change can override for service correctly.